Gemfile.lock

```diff
@@ -1,40 +1,40 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (5.0.0.beta3)
-      actionpack (= 5.0.0.beta3)
+    actioncable (5.0.0.rc1)
```

Comment:
UPGRADE NOTE: Do not worry about these changes. Your `Gemfile.lock` will update when you run bundle.